ivy/func_wrapper.py

```diff
@@ -105,23 +105,16 @@ def new_fn(*args, **kwargs):
         -------
             The return of the function, with native arrays passed in the arguments.
         """
-        if not ivy.get_array_mode():
+        if not any(isinstance(arg, ivy.Array) for arg in args) and not any(isinstance(arg, ivy.Array) for arg in kwargs.values()):
```

Comment:
I don't think we should be removing the `get_array_mode` usage as it's primary purpose is to avoid doing additional checks when Ivy isn't operated in array mode and removing this would prevent the users from having any control over this

Reply:
Just for reference, it was added here https://github.com/unifyai/ivy/pull/2276